controllers/util/mw_util.go

```diff
@@ -228,18 +228,18 @@ func ExtractMModeFromManifestWork(mw *ocmworkv1.ManifestWork) (*rmn.MaintenanceM
 
 // NetworkFence MW creation
 func (mwu *MWUtil) CreateOrUpdateNFManifestWork(
-	name, namespace, homeCluster string,
+	name, homeCluster string,
 	nf csiaddonsv1alpha1.NetworkFence, annotations map[string]string,
 ) error {
-	manifestWork, err := mwu.generateNFManifestWork(name, namespace, homeCluster, nf, annotations)
+	manifestWork, err := mwu.generateNFManifestWork(name, homeCluster, nf, annotations)
```

Comment:
We don't have a test for this? If we changed this, a test should fail.

Reply:
@nirs I suppose, if test is not failing - it doesn't exist